pkg/systemd/overview-cards/motdCard.jsx

```diff
@@ -65,9 +68,15 @@ const MotdEditDialog = ({ text }) => {
                        </Button>
                    </>
                }>
-            <TextArea resizeOrientation="vertical"
-                      value={value}
-                      onChange={setValue} />
+
+            <Stack hasGutter>
+                {error &&
+                <ModalError dialogError={error}
+                             dialogErrorDetail={errorDetail} />}
```

Comment:
These 2 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17221-20221203-085317-c9401003-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)